pkgs/by-name/fl/flashlabel-yxwl/package.nix

```diff
@@ -0,0 +1,87 @@
+{
+  autoPatchelfHook,
+  cups,
+  lib,
+  requireFile,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Not a blocker, but you can use finalAttrs syntax instead of rec, see other uses in nixpkgs